reconcile/utils/oc.py

```diff
@@ -63,6 +64,55 @@ class JobNotRunningError(Exception):
 
 
 class OC:
+    class _Decorators:
+        @classmethod
+        def process_reconcile_time(cls, function):
```

Comment:
Not a big fan of the nested class, but I lack technical arguments against it. Anyway, why not have the function in the module root?

Reply:
Hmmm, since this was so related to this class I thought it was a good idea. I don't have any preference for it to be inside the class or in the module root. I do agree that it is not pretty :-D

Reply:
Fixed in 69576f7